packages/storybook/index.html

```diff
@@ -7,13 +7,29 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0" />
   <title>Component Test Page</title>
   <script src="/src/index.ts" type="module"></script>
+  <!--
+    the following file is public/global.css, it will not have postcss processing
+    run on it and will be copied as is into the dist directory upon a build.
+  -->
   <link rel="stylesheet" href="/global.css" />
+  <!--
+    the following files have to be referenced individually for development but roll up into a single style.css
+    file in the built package, see comments in src/index.ts for more info and to control the files bundled.
+    These files are processed by postcss.
+
+    -- need to validate this statement as chrome and safari seem like they may handle them the same. The files
+    -- through vite definitely look like javascript
```

Comment:
Here is what `src/another.css` looks like when accessed through the browser while `yarn watch:vite` is running:

```
import { createHotContext as __vite__createHotContext } from "/@vite/client";import.meta.hot = __vite__createHotContext("/src/another.css");import { updateStyle as __vite__updateStyle, removeStyle as __vite__removeStyle } from "/@vite/client"
const __vite__id = "/Users/cjohnson/Projects/phase2/outline-templates/packages/storybook/src/another.css"
const __vite__css = "a.special-link{\n    font-size:24pt;\n  }"
__vite__updateStyle(__vite__id, __vite__css)
import.meta.hot.accept()
import.meta.hot.prune(() => __vite__removeStyle(__vite__id))
```